pkg/webview/view.swagger.json

```diff
@@ -156,6 +156,10 @@
         "fieldsV1": {
           "$ref": "#/definitions/v1FieldsV1",
           "title": "FieldsV1 holds the first JSON version format as described in the \"FieldsV1\" type.\n+optional"
+        },
```

Comment:
it looks like the changes in this file are because `make proto` wasn't run in #4990